src/pages/merge/bro/index.tsx

```diff
@@ -15,6 +17,16 @@ const BroMergePage = () => {
   const { data: broContractBalance } = useTokenBalance(CONTRACT_ADDRESSES.broEater, CONTRACT_ADDRESSES.wewe);
   const { data: bbroContractBalance } = useTokenBalance(CONTRACT_ADDRESSES.bbroEater, CONTRACT_ADDRESSES.wewe);
 
+  const getDaysRemaining = () => {
```

Comment:
Based on the name, the function should calculate the remaining days. But it calculates the passed days. In UI we show countdown, so it we can change the function or just subtract it from 60.